reviews/ethblock.art/documentation.md

```diff
@@ -0,0 +1,79 @@
+# Documentation
+## Created by: [kjr217](https://github.com/kjr217)
+
+## Summary
+* Address the issues provided in the file review section
+* Lots of natspec missing and unorthodoxically used throughout.
+
+## Scope
+```
+├── README.md
+└── contracts
+    ├── BArt.sol
+    ├── BFactory.sol
+    ├── BStyle.sol
+    ├── ERC721Ref.sol
+    ├── ERC721Style.sol
+```
+
+## File Review
+### `README.md`
+* Overall, readme is quite useful.
+* In BArt section, supplyPerBlock and supplyPerStyle misspelling of 'giver', should be 'given'?
+* In BArt section, artToValue misspelling of 'calculatin' should be 'calculating'
+* In BFactory, for the last 3 variable names in Store, make them consistent with the contract, i.e. in the contract they go by initials 'scfb', 'psfb', 'asl'
+* I like the flow charts, how the arrows are laid out in the Contracts flow chart could be less hectic.
+---
```

Comment:
*Maybe it would be good for diagrams to be version controlled - in order to be sure which version is actually tested